script/state.py

```diff
@@ -31,4 +35,26 @@
 except Exception as e:
     print(e)
 
-
+TIPPECANOE_DIR = '/usr/local/bin/'
+
+togeojsontiles.geojson_to_mbtiles(
+    filepaths=['./app/state.geojson'],
+    tippecanoe_dir=TIPPECANOE_DIR,
+    mbtiles_file='./app/state.mbtiles',
+    maxzoom=10
```

Comment:
do you think we should add minimum zoom as well here?

Reply:
I don't think it is necessary to add minimum zoom level because if we don't add minzoom than it will zoom from 22 to 10